Formula/v/vrc-get.rb

```diff
@@ -0,0 +1,24 @@
+class VrcGet < Formula
+  desc "Open Source alternative of Command-line client of VRChat Package Manager"
+  homepage "https://github.com/anatawa12/vrc-get"
+  url "https://github.com/anatawa12/vrc-get/archive/refs/tags/v1.7.0.tar.gz"
+  sha256 "585c3efec6ff57b03ad820da5387b63af26b54b0202d4e82dbb9d977c0d0aef3"
+  license "MIT"
+
+  head "https://github.com/anatawa12/vrc-get.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "vrc-get"
+  end
+
+  test do
+    assert_match "vrc-get", shell_output("#{bin}/vrc-get --help")
+  end
```

Comment:
Could you try with a test that covers some of its functionalities? Checks on version or help text almost never fail, even for broken installations. You may check out our [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) for some useful advice.

Reply:
In vrc-get, there are almost no commands that works without unity project and networks so it's hard to test without creating tons of files or accessing network.

In addtion, since vrc-get have no run-time file dependency (except for dylibs by OS) for working, I think there will not have broken installation unless build is broken.

In internal tests in vrc-get, threre are limited files are created but it's not guaranteed to work with partial projects.

Reply:
Should I add tests depends on the network access or test with creating tons of files?

Reply:
Network access is fine, but testing a failure to find a unity project is also fine.

Reply:
I added test checks downloading repository is fine.